herbert_core/utilities/strings/stringmatchi_log.m

```diff
@@ -0,0 +1,60 @@
+function status = stringmatchi_log (str, strcell, exact)
+% Logical array of matches or unambiguous abbreviations in a cellstr
```

Comment:
Significantly refactored function since this review. I try to follow the Matlab guideline of the first text line being 80 characters or less, as this line can be used to made an index of functions by some Matlab function. THis can result in some overly terse description. What I have done now is add Further text on the second comment line and onwards that hopefully explains better

Reply:
@tgperring Terseness may be good for the main comment but this function is not a logical array. It returns one.

Reply:
I changed the first line as well as adding a more detailed description in the second comment line onwards